src/lib/RpcApi.ts

```diff
@@ -83,8 +83,9 @@ export default class RpcApi {
     }
 
     public reject(error: Error) {
-        const ignoredErrors = [ ERROR_CANCELED, 'Request aborted' ];
-        if (ignoredErrors.indexOf(error.message) < 0) {
+        const ignoredErrorTypes = [ Errors.Types.INVALID_REQUEST.toString() ];
+        const ignoredErrors = [ ERROR_CANCELED, 'Request aborted', 'Account ID not found', 'Address not found' ];
+        if (!(ignoredErrorTypes.indexOf(error.name) >= 0 || ignoredErrors.indexOf(error.message) >= 0)) {
```

Comment:
Now that I think about it, this condition can also be written as
```suggestion
        if (ignoredErrorTypes.indexOf(error.name) < 0 && ignoredErrors.indexOf(error.message) < 0) {
```
It means to only run the capture-exception code, if both ignored-lists are not triggered.